app/src/main/java/ch/epfl/favo/view/tabs/favorList/FavorViewHolder.java

```diff
@@ -1,28 +1,51 @@
 package ch.epfl.favo.view.tabs.favorList;
 
+import android.content.Context;
+import android.view.MenuItem;
 import android.view.View;
+import android.widget.PopupMenu;
 import android.widget.TextView;
 
 import androidx.annotation.NonNull;
 import androidx.recyclerview.widget.RecyclerView;
 
+import java.util.concurrent.CompletableFuture;
+
 import ch.epfl.favo.R;
 import ch.epfl.favo.favor.Favor;
 
 class FavorViewHolder extends RecyclerView.ViewHolder {
 
   private TextView mTitleView;
   private TextView mDescriptionView;
+  private TextView buttonViewOption;
 
   FavorViewHolder(@NonNull View itemView) {
     super(itemView);
 
     mTitleView = itemView.findViewById(R.id.item_title);
     mDescriptionView = itemView.findViewById(R.id.item_desc);
+    buttonViewOption = itemView.findViewById(R.id.item_menu_btn);
   }
 
-  void bind(@NonNull Favor favor) {
+  void bind(Context context, @NonNull Favor favor) {
     mTitleView.setText(favor.getTitle());
     mDescriptionView.setText(String.valueOf(favor.getDescription()));
+    buttonViewOption.setOnClickListener(view -> {
+      PopupMenu popup = new PopupMenu(context, buttonViewOption);
+      popup.inflate(R.menu.item_menu);
+      popup.setOnMenuItemClickListener(item -> false);
+//        switch (item.getItemId()) {
+//          case R.id.item_menu_view:
+//            break;
+//          case R.id.item_menu_edit:
+//            break;
+//          case R.id.item_menu_cancel:
+//            break;
+//        }
+//        return false;
+//      });
```

Comment:
We can remove this commented block if not useful anymore

Reply:
This is for next sprint, where I actually implement those functionalities.